src/osd/PGLog.h

```diff
@@ -795,7 +795,7 @@ struct PGLog : DoutPrefixProvider {
       ldpp_dout(dpp, 10) << __func__ << ": more recent entry found: "
 			 << *objiter->second << ", already merged" << dendl;
 
-      assert(objiter->second->version > last_divergent_update);
+      assert(objiter->second->version >= last_divergent_update);
```

Comment:
@hjwsm1989 (NIT) Please consider to keep PR title < 50 characters for future PRs.
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits